### DISCUSSION TITLE
Welcome to FYSETC-CTP40 Discussions!

### DISCUSSION THREAD

**Discussion by GerogeFu** [FYSETC/FYSETC-CTP40#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tinker-devdav** [comment]
Hi, I managed to get this working properly on Pi4 with octodash and using the octopi lite build.  I mande changes to the README.MD if you can give me access to push up my brach, you can see the change
